### PR TITLE
fix: respect `stats` from the config for webpack@4

### LINKED ISSUE #2052: stats option is not respected in watch mode in 4.2.0
**Describe the bug**

Specifiying the stats option in webpack config has no impact on cli output with webpack-cli 4.2.0. Everything is always output. 
```
// webpack.config.js
module.exports = {
  ...whatever
  stats: 'none',
};
```
Using 4.1.0 outputs what I specify in the stats option.

**Expected behavior**

I expect the stats option to be respected.

**Screenshots**

4.2.0 (a lot more but i don't think a large image helps):
image

4.1.0:
image

**Additional context**

`"webpack": "^4.44.2"
`

### PR DESCRIPTION
**Pull request by evilebottnawi**
**What kind of change does this PR introduce?**

bugfix

**Did you add tests for your changes?**

Yes

**If relevant, did you update the documentation?**

No

**Summary**

fixes #2052

**Does this PR introduce a breaking change?**

No

**Other information**

webpack@4 doesn't return `compier` when `watch: true` in the config, fixed in webpack@5

### CODE REVIEW

File: packages/webpack-cli/lib/webpack-cli.js [hunk 1]
```diff
@@ -641,11 +641,22 @@ class WebpackCLI {
                     process.exit(2);
                 }
             } else {
-                logger.raw(`${stats.toString(foundStats)}`);
+                const printedStats = stats.toString(foundStats);
```
Comment (evilebottnawi) [inline packages/webpack-cli/lib/webpack-cli.js:644]: When we use `stats: 'none'` we have empty string, so no need to log empty string

File: packages/webpack-cli/lib/webpack-cli.js [hunk 2]
```diff
@@ -641,11 +641,22 @@ class WebpackCLI {
...
+                // Avoid extra empty line when `stats: 'none'`
+                if (printedStats) {
+                    logger.raw(`${stats.toString(foundStats)}`);
+                }
             }
         };
 
         compiler = this.createCompiler(options, callback);
+
+        // TODO webpack@4 return Watching and MultiWathing instad Compiler and MultiCompiler, remove this after drop webpack@4
```
Comment (anshumanv) [inline packages/webpack-cli/lib/webpack-cli.js:655]:
```suggestion
        // TODO webpack@4 return Watching and MultiWathing instead Compiler and MultiCompiler, remove this after drop webpack@4
```

**Comment by anshumanv** [comment]
lets fix CI

**Comment by evilebottnawi** [comment]
Fixed

**Comment by snitin315** [comment]
Windows 😞

**Review by snitin315 — APPROVED** [review]

**Pull request merged**